.circleci/config.yml

```diff
@@ -176,14 +176,14 @@ jobs:
             cd maven-java
             mvn -B archetype:generate -DgroupId=com.example -DartifactId=counter-value-entity -DarchetypeGroupId=com.akkaserverless -DarchetypeArtifactId=akkaserverless-maven-archetype -DarchetypeVersion=$SDK_VERSION
             cd counter-value-entity
-            mvn -B -Dakkaserverless-sdk.version=$SDK_VERSION compile
```

Comment:
we don't need this anymore since #154, the archetype comes with the right version for the sdk